app/controllers/file_submissions_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class FileSubmissionsController < ApplicationController
+  include ApplicationModule
+
+  before_action :deny_if_not_admin, :fetch_file_submission_by_id, only: %i[submitted_bundle]
+
+  def submitted_bundle
+    @file_submission = FileSubmission.create
+    @challenge = Challenge.create
+    @file_submission.challenge_id = @challenge.name
+    download_file(@file_submission.submitted_bundle, @file_submission.challenge_id)
```

Comment:
I'm not sure what this is intended to do, it seems to create a brand new (empty) file submission and then tries to download it.